backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -283,6 +290,14 @@ public Parcel PendingUpdate(Parcel parcel)
             parcel.ThrowIfNotAllowedToEdit(nameof(parcel), this.User, new[] { Permissions.PropertyEdit, Permissions.AdminProperties });
             var isAdmin = this.User.HasPermission(Permissions.AdminProperties);
 
+            parcel.PropertyTypeId = (int)(parcel.Parcels.Count > 0 ? PropertyTypes.Subdivision : PropertyTypes.Land);
```

Comment:
Need the same logic for updates. the pid on the subdivision parcel should always be replaced by whatever is on the parent parcel.